City-of-Helsinki/helsinki-paatokset
# PP-0: Add URL and image URL to trustee JSON.

**To test**
- Make sure you have up-to-date meeting, trustee and decisionmaker data.
- Checkout branch, run `make drush-cr`
- Check the kaupunginvaltuusto composition: https://helsinki-paatokset.docker.so/fi/ahjo_api/org_composition/02900
  - The node URLs should be included in the JSON data
- Add images to some trustees in Kaupungin: https://helsinki-paatokset.docker.so/fi/admin/content/trustees
  - Check the composition again. The image URLs should now be included also as 360x360 thumbnails
